doc/source/examples/dem/plate-discharge/plate-discharge.rst

```diff
@@ -0,0 +1,496 @@
+
+.. figure:: images/angle-areas.png
+   :alt: Angle of repose areas
+   :align: center
+   :name: plate-discharge-angle-areas
+
+   The areas where the angle of repose is calculated for the left (blue) and right (red) sides of the piles.
+
+.. note::
+   The calculated angles ignore the wall effects since all the particles in the depth of the container are taken into account.
+
+According to Zhou *et al.* [#zhou2002]_, the angle of repose from this type of configuration is calculated with the following formula:
+
+.. math::
+   \phi = 68.61 \mu_{\text{f,pp}}^{0.27} \mu_{\text{f,pw}}^{0.22} \mu_{\text{r,pp}}^{0.06} \mu_{\text{r,pw}}^{0.12} d_p^{-0.2}
+
+
+where :math:`\mu_{\text{f,pp}}` and :math:`\mu_{\text{f,pw}}` are the friction coefficients of the particle-particle and particle-wall interactions, respectively, :math:`\mu_{\text{r,pp}}` and :math:`\mu_{\text{r,pw}}` are the rolling friction coefficients, and :math:`d_p` is the particle diameter.
+
+
+The meaning of the rolling friction coefficient by the authors [#zhou2002]_ is different than found in Lethe. They express the coefficient as aa length in the `rolling friction model <../../../theory/multiphase/cfd_dem/dem.html#rolling-friction-models>`_. However, they also use the constant torque, therefore the rolling friction coefficient in Lethe as to be multiplied by the effective radius of the particle for results comparison:
+
+.. math::
+   \mu_{\text{r}}^{\text{eqt}} = \mu_{\text{r}}^{\text{lethe}}D_p
+
+.. figure:: images/angle-of-repose.png
+   :alt: Angle of repose results
+   :align: center
+   :name: plate-discharge-angle-graph
+
+   The angles of repose calculated from the simulation data. The solid lines are the angles computed from the highest particles on both side, while the shaded areas represent the angles for the left and the right.
+
+The theoretical angle of repose is 19.7°. We did not compute the mean of the angled of repose in order to compare the results with the literature since, even after 15 seconds of simulation, some particles are still falling from the top and the angles are still not oscillating around the same value. We can however state that the angles are close to the literature.
+
+Here we can see that the top angles from all simulations are in a range of around ±1.5 from the baseline case, which can be considered as a good agreement. We can clearly see a trend in the bottom angles using the ASC. The angles of repose are about 2° below the baseline and load balancing cases. It seems to be caused by the accumulation of particles at the bottom of the piles.
+
+
+----------
+References
+----------
+
+.. [#zhou2002] \Y.C. Zhou, B.H. Xu, A.B. Yu, P. Zulli, “An experimental and numerical study of the angle of repose of coarse spheres,” *Powder Technology*, vol. 125, pp. 45-54, 2002. doi: `10.1016/S0032-5910(01)00520-4 <https://doi.org/10.1016/S0032-5910(01)00520-4>`_\.
```

Comment:
In the compiled version, the ref appears weirdly.

Reply:
It is normal, I'm formatting the reference in all example on another branch